src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -545,6 +545,8 @@ TOKEN: /* SQL Keywords. prefixed with K_ to avoid name clashes */
 |   <K_WHERE:"WHERE">
 |   <K_WINDOW:"WINDOW">
 |   <K_WITH:"WITH">
+|   <K_WITH_OFFSET:"WITH OFFSET">
```

Comment:
We don't need the compound token. Just `WITH` and `ORDINALITY`/ The goal must be to define as few tokens as possible.